src/components/RightSidebar/Participants/CurrentParticipants/CurrentParticipants.vue

```diff
@@ -75,7 +76,28 @@ export default {
 		},
 	},
 
+	mounted() {
+		subscribe('user_status:status.updated', this.userStatusUpdated)
```

Comment:
didn't we said `user_status:status:updated` ?
```suggestion
		subscribe('user_status:status:updated', this.userStatusUpdated)
```

Reply:
https://github.com/nextcloud/server/blob/659bea6de2fea0656ea982dd0109ec259223ab61/core/src/views/UnifiedSearch.vue#L352

Reply:
https://github.com/nextcloud/server/pull/22526#discussion_r480920065
Maaaan, I'm so confused, let's just pick one and move on xD

Reply:
I'll move the unified search one to a dot too :sweat_smile: 
cc @raimund-schluessler

Reply:
(tomorrow, I'm too sleepy now :zzz: )

Reply:
Maybe we should write that down somewhere. Changing it to a dot requires alls apps to also adjust again (I don't know how many use this event already, though).

Reply:
https://github.com/nextcloud/nextcloud-event-bus/issues/131#issuecomment-683696800 also had a dot in the example. I mean we can also subscribe to both or something. But anyway it needs fixing in the server then I guess.

Reply:
> Maybe we should write that down somewhere. Changing it to a dot requires alls apps to also adjust again (I don't know how many use this event already, though).

not much, only you I think :)
Let's go for the dot, whatever was discussed and I misunderstood seems to not be what @ChristophWurst suggested in https://github.com/nextcloud/nextcloud-event-bus/issues/131#issuecomment-683660316

Reply:
Also updated main post https://github.com/nextcloud/nextcloud-event-bus/issues/131#issue-658951244

Reply:
Fine with me, let's move to a dot then.

Reply:
Please see https://github.com/nextcloud/server/pull/22868 @nickvergessen @raimund-schluessler